WordPress/Classes/ViewRelated/NUX/SignupDomainSuggestionViewController.swift

```diff
@@ -0,0 +1,244 @@
+        case Sections.searchSuggestions.rawValue:
+            if isSearching {
+                cell = activityCell()
+            } else {
+                cell = searchButtonCell(index: indexPath.row)
+            }
+        case Sections.siteTitleSuggestions.rawValue:
+            fallthrough
+        default:
+            cell = buttonCell(index: indexPath.row)
+        }
+        return cell
+    }
+
+    override func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        guard section == Sections.siteTitleSuggestions.rawValue || section == Sections.searchField.rawValue else {
+            return nil
+        }
+        let footer = UIView()
+        footer.backgroundColor = WPStyleGuide.greyLighten20()
+        return footer
+    }
+
+    override func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        guard section == Sections.siteTitleSuggestions.rawValue || section == Sections.searchField.rawValue else {
+            return 0.0
+        }
+        return 0.5
+    }
+
+    // MARK: table view cells
+
+    private func activityCell() -> SiteCreationDomainsActivityTableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: SiteCreationDomainsActivityTableViewCell.cellIdentifier) as? SiteCreationDomainsActivityTableViewCell else {
+            let newCell = SiteCreationDomainsActivityTableViewCell(style: .default, reuseIdentifier: SiteCreationDomainsActivityTableViewCell.cellIdentifier)
+            newCell.activitySpinner?.startAnimating()
+            return newCell
+        }
+        cell.activitySpinner?.startAnimating()
+        return cell
+    }
+
+    private func titleAndDescriptionCell() -> UITableViewCell {
+        let title = NSLocalizedString("Step 4 of 4", comment: "Title for last step in the site creation process.").localizedUppercase
+        let description = NSLocalizedString("Pick an available \"yourname.wordpress.com\" address to let people find you on the web.", comment: "Description of how to pick a domain name during the site creation process")
+        let cell = LoginSocialErrorCell(title: title, description: description)
+        cell.selectionStyle = .none
+        return cell
+    }
+
+    private func searchFieldCell() -> SiteCreationDomainSearchTableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: SiteCreationDomainSearchTableViewCell.cellIdentifier) as? SiteCreationDomainSearchTableViewCell else {
+            return SiteCreationDomainSearchTableViewCell(placeholder: "")
+        }
+        cell.delegate = self
+        cell.selectionStyle = .none
+        return cell
+    }
+
+    private func buttonCell(index: Int) -> UITableViewCell {
+        let cell = UITableViewCell()
+
+        let suggestion = siteTitleSuggestions[index]
+
+        cell.textLabel?.text = suggestion
+        cell.textLabel?.textColor = WPStyleGuide.darkGrey()
+        return cell
+    }
```

Comment:
The cell separators don't go all the way to the left:
<img width="527" alt="screen shot 2017-12-19 at 2 35 26 pm" src="https://user-images.githubusercontent.com/1816888/34179897-f13ab1e4-e4c9-11e7-9527-47bb47d608c9.png">